update(perf): Update performance landing and widget descriptions (#4500)
getsentry/sentry-docs: src/docs/product/performance/mobile-vitals.mdx

> Mobile Vitals are summarized in several graphs on the **Performance** page in [sentry.io](https://sentry.io) when you select "Mobile" in the "Display" dropdown.

Reviewer: ```suggestion
Mobile Vitals are summarized in several graphs on the **Performance** page in [sentry.io](https://sentry.io) in the "Mobile" tab.
```
